src/types/Authorization.ts

```diff
@@ -183,7 +184,10 @@ export interface AuthorizationActions {
   canRead?: boolean;
   canCreate?: boolean;
   canUpdate?: boolean;
+  canUpdateByVisualID?: boolean;
   canDelete?: boolean;
+  canUnassign?: boolean;
+  canAssign?: boolean;
```

Comment:
Create a new type TagAuthorizationActions and move the 3 canXXX inside.

Reply:
Done :)